src/sidebar/services/api.ts

```diff
@@ -171,6 +171,10 @@ type ListGroupParams = {
   expand?: string[];
 };
 
+type AnalyticsEvent = {
+  event: 'APPLY_PENDING_UPDATES';
```

Comment:
I think we'll want to come up with some scheme for organizing events so we can easily find related events. For example, something like `{app}.{area}.{event}`, `client.realtime.apply_updates`.

Reply:
I agree with the general idea, but this requires changing the BE, as we added some schema validation to avoid anyone from abusing this endpoint and flooding our logs with nonsense (see https://github.com/hypothesis/h/pull/8623)

I'll create another PR there before merging this, to change the schema it validates.

Reply:
I have just created https://github.com/hypothesis/h/pull/8643. Once that's merged I will update this PR and request a new review